client/src/test/java/io/spine/core/CommandsTest.java

```diff
@@ -117,115 +142,129 @@ public void check_if_contexts_have_same_actor_and_tenantId() {
     }
 
     @Test
-    public void pass_null_tolerance_test() {
-        new NullPointerTester()
-                .setDefault(FileDescriptor.class, DEFAULT_FILE_DESCRIPTOR)
-                .setDefault(Timestamp.class, getCurrentTime())
-                .setDefault(Duration.class, Durations2.ZERO)
-                .setDefault(Command.class,
-                            requestFactory.createCommand(StringValue.getDefaultInstance(),
-                                                         minutesAgo(1)))
-                .setDefault(CommandContext.class, requestFactory.createCommandContext())
-                .setDefault(ZoneOffset.class, ZoneOffsets.UTC)
-                .setDefault(UserId.class, GivenUserId.newUuid())
-                .testStaticMethods(Commands.class, NullPointerTester.Visibility.PACKAGE);
-    }
-
-    @Test
-    public void generate_command_ids() {
+    @DisplayName("generate command id")
+    void generateCommandId() {
         final CommandId id = Commands.generateId();
 
         assertFalse(Identifier.toString(id)
                               .isEmpty());
     }
 
     @Test
-    public void extract_message_from_command() {
+    @DisplayName("extract message from given command")
+    void extractMessage() {
         final StringValue message = toMessage("extract_message_from_command");
 
         final Command command = requestFactory.createCommand(message);
         assertEquals(message, Commands.getMessage(command));
     }
 
-    @Test
-    public void create_wereAfter_predicate() {
-        final Command command = requestFactory.command()
-                                              .create(BoolValue.getDefaultInstance());
-        assertTrue(Commands.wereAfter(secondsAgo(5))
-                           .apply(command));
+    @Nested
+    @DisplayName("when creating predicate for commands")
+    class CreatePredicateTest {
+
+        @Test
+        @DisplayName("support `wereAfter` predicate")
+        void createWereAfterPredicate() {
+            final Command command = requestFactory.command()
+                                                  .create(BoolValue.getDefaultInstance());
+            assertTrue(Commands.wereAfter(secondsAgo(5))
+                               .apply(command));
+        }
+
+        @Test
+        @DisplayName("support `wereBetween` predicate")
+        void createWereBetweenPredicate() {
+            final Command command1 = requestFactory.createCommand(StringValue.getDefaultInstance(),
```

Comment:
Let's use this moment and rename these local declarations to `fiveMinsAgo`, `twoMinsAgo` etc.